Sources/UIKit/UIView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  UIView.swift
+//  Alicerce
+//
+//  Created by Tiago Veloso on 15/02/2018.
+//  Copyright © 2018 Mindera. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIView {
+
+    /// Similar to NSLayoutConstraint.activate(), but this method ensures that the views
+    /// that are adding the constraints disable translatesAutoresizingMaskIntoConstraints
+    ///
+    /// - Parameter constraints: an array of constrains to activate
+    public static func activateConstraints(_ constraints: [NSLayoutConstraint]) {
+
+        constraints.forEach {
+
+            if let view = $0.firstItem as? UIView {
+
+                view.translatesAutoresizingMaskIntoConstraints = false
+            }
+        }
+
+        NSLayoutConstraint.activate(constraints)
+    }
+
+    func edgesToView(_ view: UIView, insets: UIEdgeInsets = .zero) {
```

Comment:
perhaps a better name could be `constrainEdgesToView(_ view: UIView, insets: UIEdgeInsets = .zero)`?